sdk/basyx/__init__.py

```diff
@@ -0,0 +1 @@
+from .objectstore import *
```

Comment:
Based on the Python naming style, I'd call this module `object_store` instead of `objectstore`